app/webpacker/components/ContactEditProfilePage/EditProfileForm.jsx

```diff
@@ -102,13 +100,10 @@ export default function EditProfileForm({
         onChange={handleFormChange}
         required
       />
-      <Form.Select
-        options={countryOptions}
+      <CountrySelector
         label={i18n.t('activerecord.attributes.user.country_iso2')}
-        name="country_iso2"
-        search
-        value={editedProfileDetails?.country_iso2}
-        onChange={handleFormChange}
+        selected={editedProfileDetails?.country_iso2}
+        onChange={(event, data) => handleFormChange(null, { name: 'country_iso2', value: data.value })}
```

Comment:
If you keep passing in `name` (not sure why you deleted that in the first place), then just keeping `onChange={handleFormChange}` should work